### PR TITLE
[4.0] readmore with php7.4

### PR DESCRIPTION
**Pull request by brianteeman**
Pull request for #27239

Fixes a b/c breaking change from php 7.4

### Steps to reproduce the issue
Create a new article with full php errors displayed **_using php7.4_**

### Actual result
Notice: Trying to access array offset on value of type null in C:\htdocs\joomla-cms\plugins\editors-xtd\readmore\readmore.php on line 52

After applying the PR the notice is gone and readmore plugin works normally

### CODE REVIEW

File: plugins/editors-xtd/readmore/readmore.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ public function onDisplay($name)
 		);
 
 		$getContentResult = $this->getDispatcher()->dispatch('getContent', $event);
-		$getContent = $getContentResult['result'][0];
+		$getContent = isset($getContentResult['result'][0]);
```
Comment (Quy) [inline plugins/editors-xtd/readmore/readmore.php:52]: I don't know what value it should be, however, this will make it true/false.

Comment (SharkyKZ) [reply plugins/editors-xtd/readmore/readmore.php:52]: In 3.x it's editor field's ID. In 4.0 it looks like it's not being used anymore https://github.com/joomla/joomla-cms/blob/c80a4a0c1adb5027de5da9da4ecef89d60129cb4/build/media_source/com_content/js/admin-article-readmore.es6.js.

Comment (brianteeman) [reply plugins/editors-xtd/readmore/readmore.php:52]: so shall I just kill it?

Comment (SharkyKZ) [reply plugins/editors-xtd/readmore/readmore.php:52]: I guess so, assuming related JS code was removed intentionally.

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 007d5a45db66a0494d6f36c43d6928348b37f1f8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27327">issues.joomla.org/tracker/joomla-cms/27327</a>.</sub>

File: plugins/editors-xtd/readmore/readmore.php (hunk 1 above)
Comment (brianteeman) [reply plugins/editors-xtd/readmore/readmore.php:52]: @SharkyKZ If I remove that line then I get an error that getConstantResult is not defined

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 22cd67c474739985cd1e6f395eb2b40163720510<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27327">issues.joomla.org/tracker/joomla-cms/27327</a>.</sub>

**Comment by Fedik** [comment]
A whole `ondDisplay` are wrong. The thing that there no event `getContent`:
`$getContentResult = $this->getDispatcher()->dispatch('getContent', $event);`

`getContent` is not an event, and never was, it was editor "method", compare to Joomla 3:

https://github.com/joomla/joomla-cms/blob/2d1ea696e9d52818ce8f34516d2b5524148546f0/plugins/editors-xtd/readmore/readmore.php#L41-L47
there we have `$this->_subject` that is Editor instance, if I right remember.

Need to review whole plugin.

**Comment by no-matter-0** [comment]
same error here

**Comment by Quy** [comment]
Closing per PR #28401<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27327">issues.joomla.org/tracker/joomla-cms/27327</a>.</sub>

**Pull request closed without merging**